### DISCUSSION TITLE
I can't get the env variables to load

### DISCUSSION THREAD

**Discussion by EphraimB** [mikro-orm/mikro-orm#5866] [General]
### Describe the bug

I know that it mentions [here](https://mikro-orm.io/docs/upgrading-v5-to-v6) that I need to manually load the env variables using dotenv.register() but I then get a type error that register doesn't exist.

### Reproduction

https://github.com/EphraimB/budgeting/tree/replace-with-nestjs

### What driver are you using?

@mikro-orm/postgresql

### MikroORM version

6.3.0

### Node.js version

20

### Operating system

Docker

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/mikro-orm/mikro-orm/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://mikro-orm.io/docs).
- [X] Check that there isn't [already an issue](https://github.com/mikro-orm/mikro-orm/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/mikro-orm/mikro-orm/discussions) or join our [Discord](https://discord.gg/w8bjxFHS7X).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by B4nan** [comment]
> I know that it mentions [here](https://mikro-orm.io/docs/upgrading-v5-to-v6) that I need to manually load the env variables using dotenv.register() but I then get a type error that register doesn't exist.

If you have problems with dotenv, you should ask their support, this is not really related to the ORM. 

> Reproduction

This is not a reproduction, it's a huge app on its own, I can't help with that. But from a quick look, I don't even see you would have dotenv installed anywhere explicitly.

**Reply by EphraimB** [reply]
The documentation says dotenv.register() but no such function exists. That's what dotenv support said.

**Reply by B4nan** [reply]
Oh I see, you should use whatever they say to register it. Looking at their docs, it should be either `import 'dotenv/config'` or `require('dotenv').config()`, will adjust the upgrading guide.

**Comment by EphraimB** [comment]
But even that doesn’t work

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
